steamtinkerlaunch

```diff
@@ -618,6 +618,12 @@ function setSteamPath {
 }
 
 function setSteamPaths {
+	if [ ! -x "$(command -v "steam")" ]; then
```

Comment:
```suggestion
	if [ ! -x "$(command -v "steam")" ] && [ "$INFLATPAK" -eq 0 ]; then
```

Ensures we don't fail with missing Steam when running in Flatpak, where `steam` may still work, but I'd rather not do this kind of check (Flatpak STL can't really be used outside of Flatpak Steam to begin with, so we wouldn't need to check for Steam there anyway).